components/list/list-item-mixin.js

```diff
@@ -60,6 +60,10 @@ export const ListItemMixin = superclass => class extends composeMixins(
 			 * @type {string}
 			 */
 			color: { type: String },
+			/**
+			 * @ignore
+			 */
+			first: { type: Boolean, reflect: true },
```

Comment:
I added a `first` property to be set by the parent list on the first list item of a given list